### PR TITLE
Add section about unit tests

### PR DESCRIPTION
**Pull request by erthalion**
For #226 

Documents how to run all the unit tests for different golang versions and how to debug them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,3 +350,27 @@ kubectl port-forward POD_NAME DLV_PORT:DLV_PORT
...
+
+```
+go test ./...
+```
+
+For go 1.9 `vendor` directory would be excluded automatically. For previous
+versions you can exclude it manually:
+
+```
+go test (glide novendor)
```
Comment (mgomezch) [inline README.md]: Missed a `$`?

**Comment by zerg-junior** [comment]
:+1:

**Comment by erthalion** [comment]
:+1:

**Comment by mgomezch** [comment]
:+1:

**Pull request merged**